webcomponents/tool/src/main/frontend/js/calendar/sakai-calendar.js

```diff
@@ -73,13 +73,13 @@ export class SakaiCalendar extends LionCalendar {
 
     this.shadowRoot.querySelectorAll(".calendar__day-button").forEach(c => {
 
-      c.classList.remove("has-event");
+      c.classList.remove("has-event", "sakai-colorize-green-7");
       c.classList.remove("deadline");
 
       let time = c.date.getTime();
       let e = this.events.find(e => e.start > time && e.start < (time + 24*60*60*1000));
       if (e) {
-        c.classList.add("has-event");
+        c.classList.add("has-event", "sakai-colorize-green-7");
```

Comment:
This won't work in shadowdom. You have to modify the has-event class to use that colour, as a css var. Have we got a mnemonic for sakai-colorize-green-7? Like 'sakai-ok', or whatever?